### ISSUE TITLE
failed

### ISSUE THREAD

**Issue by 0d9y** [OleksandrDoskaliuk/hadoop-hive-spark#1]
historyserver                | [8/100] check for namenode:9000...
historyserver                | [8/100] namenode:9000 is not available yet

Have you tested?

**Comment by OleksandrDoskaliuk** [comment]
Yes, I have tested this cluster on three machines (2 windows and Ubuntu). And it works fine

**Comment by 0d9y** [comment]
centos is failed
